### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by pchico83**
This branch will have all the changes needed for the Okteto 1.0 release

### CODE REVIEW

**Review by RinkiyaKeDad — APPROVED** [review]

**Comment by jmacelroy** [comment]
@pchico83 I don't know that we're ready for this are we? We still need the release notes which were blocked by the commit from Adrian until recently. Also, we need https://www.notion.so/okteto/Okteto-1-0-release-a8446057f7e74a678f3238294d082ebd#62b7ffc04b854671a9b8104c7f39a951 which @huguestennier is looking into. 

We can start reviews on this but it's not ready and I don't expect to merge until after OE release is live. Does this work?

**Comment by huguestennier** [comment]
Flipped `/enterprise` to `/self-hosted` and fixed a bunch of broken links while i was there

File: src/content/self-hosted/eks.mdx [hunk 1]
```diff
@@ -31,9 +31,10 @@ We recommend that you follow Amazon's [cluster creation guide](https://docs.aws.
 Okteto supports Kubernetes versions 1.20 through 1.22.
 
 We recommend the following specs:
+
 - v1.22
-- A pool with at least 3 `m4.xlarge` nodes
-- 100 GB per disk
+- A pool with at least 3 nodes (4CPUs and 32GBs each)
```
Comment (rberrelleza) [inline src/content/self-hosted/eks.mdx:36]: I think this is a lot, and might scare people away. Do you think we can give guidance based on the expected size of the team?

Comment (rberrelleza) [reply src/content/self-hosted/eks.mdx:36]: or maybe give both minimum and recommended ?

File: src/content/self-hosted/eks.mdx [hunk 2]
```diff
@@ -138,22 +124,18 @@ helm repo update
 
 ### Getting your Okteto License
 
-Okteto is free for small teams.  You get all the features of [Okteto](/docs/enterprise/) for up to 3 users with 3 namespaces each.
+Okteto is free for small teams. You get all the features of [Okteto](/docs/self-hosted/) for up to 3 users with 3 namespaces each.
```
Comment (rberrelleza) [inline src/content/self-hosted/eks.mdx:127]: I think we should send them to https://okteto.com/free-trial instead, but double check with @bigash42

File: src/content/self-hosted/eks.mdx [hunk 3]
```diff
@@ -258,7 +241,7 @@ NAME                             TYPE           CLUSTER-IP   EXTERNAL-IP
 okteto-ingress-nginx-controller  LoadBalancer   10.0.7.73    a519c8b3b27f94...elb.amazonaws.com   80:30795/TCP,443:32481/TCP,1234:30885/TCP   5m
 ```
 
-You'll need to take the `EXTERNAL-IP` address, and add it to your DNS for the domain you have chosen to use. In Route 53, this is done by creating an `A` record with the name  `*`, pointing to the alias of the Elastic Load Balancer.
+You'll need to take the `EXTERNAL-IP` address, and add it to your DNS for the domain you have chosen to use. In Route 53, this is done by creating an `A` record with the name `*`, pointing to the alias of the Elastic Load Balancer.
```
Comment (rberrelleza) [inline src/content/self-hosted/eks.mdx:244]: in some parts we call it subdomain, in others we call it domain. we should be consistent.

File: src/content/self-hosted/eks.mdx [hunk 4]
```diff
@@ -273,21 +256,14 @@ kubectl get service -l=app.kubernetes.io/instance=okteto,app.kubernetes.io/compo
...
 ```
-NAME                                  TYPE           CLUSTER-IP      EXTERNAL-IP                           PORT(S)          AGE
-okteto-okteto-enterprise-buildkit     LoadBalancer   10.245.142.73   a519c8b3b27f95...elb.amazonaws.com    1234:32449/TCP   5m
+NAME                TYPE           CLUSTER-IP      EXTERNAL-IP                           PORT(S)          AGE
+okteto-buildkit     LoadBalancer   10.245.142.73   a519c8b3b27f95...elb.amazonaws.com    1234:32449/TCP   5m
 ```
 
 You'll need to take the `EXTERNAL-IP` address and add it to your DNS for the domain you have chosen to use. This is done by creating an `A` record with the name `buildkit`.
 
 ## Sign in to your Okteto
```
Comment (rberrelleza) [inline src/content/self-hosted/eks.mdx:265]:
```suggestion
## Sign in to your Okteto instance
```

File: src/content/self-hosted/eks.mdx [hunk 5]
```diff
@@ -273,21 +256,14 @@ kubectl get service -l=app.kubernetes.io/instance=okteto,app.kubernetes.io/compo
...
 
-You can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as you soon as you click the `Login` button.
-
-## Troubleshooting
+> **Important**: The Okteto default installation uses self-signed certificates. We strongly recommend you to reconfigure Okteto to [bring your own certificates](/docs/self-hosted/administration/certificates/) next.
```
Comment (rberrelleza) [inline src/content/self-hosted/eks.mdx:267]:
```suggestion
> **Important**: Okteto's default installation uses self-signed certificates. We strongly recommend that you  configure Okteto to [bring your own certificates](/docs/self-hosted/administration/certificates/) before giving your team access to your instance.
```

File: src/content/self-hosted/eks.mdx [hunk 6]
```diff
@@ -273,21 +256,14 @@ kubectl get service -l=app.kubernetes.io/instance=okteto,app.kubernetes.io/compo
...
 
-If you experience issues during this guide, we recommend you check these things first:
-
-1. `okteto.SUBDOMAIN` resolves to the IP address you retrieved.
-1. If you get a certificate warning, there has been a problem with Let's Encrypt. This is usually related to DNS resolution, the need to retry, or the permissions of the credentials created.
-
-For further troubleshooting tips, see our [troubleshooting guide](/docs/enterprise/install/troubleshooting/).
+You can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as you soon as you click the `Login` button.
```
Comment (rberrelleza) [inline src/content/self-hosted/eks.mdx:269]:
```suggestion
You can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as part of the login process. The first user to successfully log into the instance will be automatically assigned the `administrator` role.
```

File: src/content/self-hosted/eks.mdx (hunk 1 above)
Comment (pchico83) [reply src/content/self-hosted/eks.mdx:36]: I will undo those changes to reduce moving parts in this release. After the release, I would like to keep iterating on installation guides

**Pull request merged**